addons/mail/static/src/models/channel.js

```diff
@@ -127,6 +127,17 @@ registerModel({
             }
             return `${window.location.origin}/chat/${this.id}/${this.uuid}`;
         },
+        /**
+        * @private
+        * @returns {boolean}
+        */
+        _computeIsDescriptionEditable() {
+            const typesOfChannelWithEditableDescription = new Set([
+                'channel',
+                'group',
+            ]);
+            return typesOfChannelWithEditableDescription.has(this.channel_type);
```

Comment:
you don't gain anything with `new Set().has()` rather than `[].includes()`. Actually it less performant this code, because compute creates a set and its quite a costly LOC

Reply:
I didn't choose the Set for performance reasons, it just seemed to better convey the idea of the code. But if you think that's a problem, I can go back to `includes`, sure